docker/docker-entrypoint.sh

```diff
@@ -1,17 +1,17 @@
 #!/bin/bash -xe
 
-if [[ -z  ${REMOTE_NETTY_TCP_PORT} ]]; then
-  echo "Error: env variable REMOTE_NETTY_TCP_PORT is not defined." 
+if [[ -z  ${PORT} ]]; then
+        export PORT=2551
```

Comment:
I think that it could be better assign 2551 to a DEFAULT_PORT = 2551 at the beginning of the script.
Then inside the if you could put PORT = DEFAULT_PORT